project.json

```diff
@@ -6,5 +6,12 @@
   "Package": "plugin",
   "Executable": "",
   "AppName": "",
-  "AppUsage": ""
+  "AppUsage": "",
+  "DockerOwner": "plugins",
+  "DockerRepo": "",
+  "DockerMaintainer": "Drone.IO Community <drone-dev@googlegroups.com>",
+  "DockerSchemaName": "",
+  "DockerSchemaVendor": "Drone.IO Community",
+  "DockerSchemaVersion": "1.0",
```

Comment:
I think it doesn't make sense to have the schema version configurable as it's a static value.

Reply:
Fixed